main.nf

```diff
@@ -311,17 +338,19 @@ process miRBasePostAlignment {
     maxRetries 3
     maxErrors '-1'
 
-    publishDir "${params.outdir}/miRBaseCounts", mode: 'copy'
+    publishDir "${params.out}", mode: 'copy', saveAs: this.&rule_step5
```

Comment:
`${params.out}` not `${params.outdir}`? Is this actually used at all? If not might be nice to make that clear:
```groovy
publishDir 'set_by_rule', mode: 'copy', saveAs: this.&rule_step5
```